### PR TITLE
Support nullif

### PR DESCRIPTION
**Pull request by MrLolthe1st**

### CODE REVIEW

File: ydb/library/yql/tests/sql/suites/pg/nullif.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+--!syntax_pg
+select nullif(1, 1) as x, nullif(2, 1) as y;
```
Comment (vitstn) [inline ydb/library/yql/tests/sql/suites/pg/nullif.sql:2]: please add a test with NULL

File: ydb/library/yql/sql/pg/pg_sql.cpp [hunk 2]
```diff
@@ -4380,6 +4380,27 @@ class TConverter : public IPGParseEvents {
...
+
+        auto lhs = ParseExpr(value->lexpr, settings);
+        auto rhs = ParseExpr(value->rexpr, settings);
+        if (!lhs || !rhs) {
+            return nullptr;
+        }
+        auto pred = L(A("Coalesce"),
+                L(A("FromPg"), L(A("PgOp"), QA("="), lhs, rhs)),
+                L(A("Bool"), QA("false")));
+        return L(A("If"), pred, lhs, L(A("Null")));
```
Comment (vitstn) [inline ydb/library/yql/sql/pg/pg_sql.cpp:4401]: type of result should be pgbool

**Review by vitstn — APPROVED** [review]
